TestingLowerBounds/Convex.lean

```diff
@@ -77,4 +77,10 @@ lemma comp_neg_iff {𝕜 F β : Type*} [LinearOrderedField 𝕜] [AddCommGroup F
   rw [← neg_neg s, ← Function.comp_id f, ← neg_comp_neg, ← Function.comp.assoc]
   exact h.comp_neg
 
+--this can be stated in much greater generality
+lemma const_mul (c : ℝ) : ConvexOn ℝ Set.univ (fun (x : ℝ) ↦ c * x) := by
```

Comment:
```suggestion
lemma const_mul_id (c : ℝ) : ConvexOn ℝ Set.univ (fun (x : ℝ) ↦ c * x) := by
```
I would expect `const_mul` to be the statement that `x ↦ c * f x` is convex (for non-negative c). `const_mul_id ` seems more fitting here.